docs/pages/reference/terraform/data-sources/access_list.mdx

```diff
@@ -0,0 +1,168 @@
+---
+title: "teleport_access_list Data Source - teleport"
+description: |-
+  
+---
+
+# teleport_access_list (Data Source)
```

Comment:
Docs pages already get an H1 from the title, so we'll want to delete the second H1 here.